node/src/chaincodeeventsrequest.ts

```diff
@@ -36,22 +36,21 @@ export interface ChaincodeEventsRequest extends Signable {
     getEvents(options?: CallOptions): Promise<CloseableAsyncIterable<ChaincodeEvent>>;
 }
 
-export interface ChaincodeEventsRequestOptions {
+export interface SignedChaincodeEventsRequestOptions {
```

Comment:
These are still the options used when creating a ChaincodeEventsRequest so the name should remain the same.

Reply:
It is already `SignedChaincodeEventsRequestOptions`